peepingtom/core/alchemists/from_points.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+
+from ..datablocks import LineBlock, ParticleBlock, PointBlock, OrientationBlock, PropertyBlock
+from ..base import Alchemist
+
+
+class PointToLineAlchemist(Alchemist):
+    """
+    transform a PointBlock into a LineBlock
+    """
+    def __init__(self, pointblock):
+        super().__init__({'points': pointblock})
```

Comment:
it doesn't feel obvious why you're passing a dictionary here to super().__init__() even after reading the Alchemist base class - more explanation needed?